packages/data/addon/models/metadata/dimension.ts

```diff
@@ -23,7 +23,7 @@ export interface DimensionMetadata extends ColumnMetadata {
 export interface DimensionMetadataPayload extends ColumnMetadataPayload {
   fields?: Field[];
   cardinality?: Cardinality;
-  storageStrategy?: 'loaded' | 'none';
+  storageStrategy?: TODO<'loaded' | 'none' | null>;
```

Comment:
Are we planning to come back to this?

Reply:
Maybe. It should at least be better defined because previously we had this as what was returned from fili
https://github.com/yahoo/navi/blob/71325fcfe781aac9bdc20f874437ff9d9bc72d0f/packages/data/addon/serializers/metadata/bard.ts#L29-L39
so `storageStrategy` was always defined, but we also had this
https://github.com/yahoo/navi/blob/c96e5c5404e4fd9d1cc097d4220d15b454809161/packages/data/addon/serializers/metadata/bard.ts#L408

so that's why I'm updating it